### PR TITLE
added draft custom harness creation doc

### PR DESCRIPTION
**Pull request by boltzmann-brain**
Per discussion, added draft topic on how to create a custom harness, assuming ideal user experience

Also fixed broken link elsewhere

### CODE REVIEW

File: source/python-sdk/structure/custom-harness.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+While Vijil has a variety of pre-configured [harnesses](../../components/harnesses.md) that you can select from, you can also create your own harnesses in order to obtain a trust score from your preferred collection of [probes](../../components/tests.md).
+
+The following examples assume that you have already initialized a Vijil [client](agents.md).
+
+## Create harness
+
+Use the `Harnesses.create` function in the Python [client](agents.md) to specify the list of probes you want to include in the harness:
+
+```python
+client.Harnesses.create(name="my custom harness", probes=["probe1", "probe2"])
```
Comment (shubhobm) [inline source/python-sdk/structure/custom-harness.md:12]: let's do `client.harnesses.*` and `client.evaluations.*`
I'll roll in changes in the python client

File: source/python-sdk/structure/custom-harness.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#   {'name': 'custom2', 'probes': ['probe4', 'probe5']},
+#   {'name': 'fairness', 'probes': ['vijil.probes.adultdata.CounterfactualGender', 'vijil.probes.winobias.ProfessionalBias']}, 
+# ...
+# ]
+```
+
+## Delete harness
+
+```python
+client.Harnesses.delete(name="my custom harness")
```
Comment (shubhobm) [inline source/python-sdk/structure/custom-harness.md:35]: `client.harnesses.*`

**Review by shubhobm — CHANGES_REQUESTED** [review]
small changes

**Comment by boltzmann-brain** [comment]
@shubhobm changed casing as requested

**Review by shubhobm — APPROVED** [review]
Lgtm

**Pull request merged**